### ISSUE TITLE
Attacking the designated user while hunting

### ISSUE THREAD

**Issue by Fatalitly** [grimaldello/the-crims-italian-bot-v2#74]
Attack by username can be added while hunting. Only the user whose name or ID is written can be given the command to attack.

**Comment by grimaldello** [comment]
Hi @Fatalitly,

thank you for your reporting.

You mean to add a feature in order to attack only a specific list of usernames/IDs?

For example:
```javascript
usernamesToAttackList: ["Greg", "Pippo"], // Attack only user Greg and Pippo
idsToAttackList: ["666", "777"] // Attack only user with id 666 and 777
```

**Comment by Fatalitly** [comment]
> MERHABA@Fatalitly,
> 
> bildirdiğiniz için teşekkür ederiz.
> 
> Yalnızca belirli bir kullanıcı adı/kimliği listesine saldırmak için bir özellik eklemek mi istiyorsunuz?
> 
> Örneğin:
> 
> ```js
> usernamesToAttackList: ["Greg", "Pippo"], // Attack only user Greg and Pippo
> idsToAttackList: ["666", "777"] // Attack only user with id 666 and 777
> ```

Yes bro

**Comment by grimaldello** [comment]
OK, it can be implemented.

**Comment by grimaldello** [comment]
Done but not tested. Please report any issues.

This functionality is available in v0.0.29 of the bot.

Explanation in the last point of: https://github.com/grimaldello/the-crims-italian-bot-v2#hunting
